warplib/maths/gt.cairo

```diff
@@ -9,8 +9,6 @@ func warp_gt{range_check_ptr}(lhs : felt, rhs : felt) -> (res : felt):
 end
 
 func warp_gt256{range_check_ptr}(lhs : Uint256, rhs : Uint256) -> (res : felt):
-    alloc_locals
-    let (local lt) = uint256_lt(lhs, rhs)
-    let (eq) = uint256_eq(lhs, rhs)
-    return ((1 - lt) * (1 - eq))
+    let (le) = uint256_le(lhs, rhs)
```

Comment:
Also tried introducing this helper in Cairo: https://github.com/starkware-libs/cairo-lang/pull/72